Your pull request does not meet our code style rules. Pylint summary:
```
************* Module lbf
samples/lbf.py:25:0: W0611: Unused import sys (unused-import)
```

You can generate these warnings with `maintainer/CI/fix_style.sh`. This is the same command that I have executed to generate the log above.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -19,7 +19,7 @@ repos:
         always_run: false
         files: '.*\.(py|pyx|pxd)'
         exclude: '\.pylintrc|.*.\.py\.in|^libs/'
-        args: ["--ignore=E266,E701,W291,W293", "--in-place", "--aggressive"]
+        args: ["--ignore=E266,E402,E701,W291,W293", "--in-place", "--aggressive"]
 
     -   id: cmake-format
         name: cmake-format

--- a/doc/tutorials/convert.py
+++ b/doc/tutorials/convert.py
@@ -61,7 +61,7 @@ def add_cell_from_script(nb, filepath):
     if m and all(x.startswith('#') for x in m.group(0).strip().split('\n')):
         code = re.sub('^(#\n)+', '', code.replace(m.group(0), ''), re.M)
     # strip first component in relative paths
-    code = re.sub('(?<=[\'\"])\.\./', './', code)
+    code = re.sub(r'(?<=[\'\"])\.\./', './', code)
     # create new cells
     filename = os.path.relpath(os.path.realpath(filepath))
     if len(filename) > len(filepath):

--- a/maintainer/CI/fix_style.sh
+++ b/maintainer/CI/fix_style.sh
@@ -30,24 +30,28 @@ pre_commit_return_code="${?}"
 
 git diff-index --quiet HEAD --
 git_diff_return_code="${?}"
-if [ "${pre_commit_return_code}" = 1 ]; then
-    if [ "${git_diff_return_code}" = 1 ]; then
-        if [ "${CI}" != "" ]; then
-            git --no-pager diff > style.patch
-            maintainer/gh_post_style_patch.py || exit 1
-            echo "Failed style check. Download ${CI_JOB_URL}/artifacts/raw/style.patch to see which changes are necessary." >&2
-        else:
-            echo "Failed style check." >&2
-        fi
+
+pylint_errors=0
+if [ -f "pylint.log" ]; then
+    pylint_errors=$(grep -Pc '^[a-z]+/.+?.py:[0-9]+:[0-9]+: [CRWEF][0-9]+:' "pylint.log")
+fi
+
+if [ "${CI}" != "" ]; then
+    git --no-pager diff > style.patch
+    maintainer/gh_post_style_patch.py || exit 1
+    maintainer/gh_post_pylint.py "${pylint_errors}" pylint.log || exit 1
+    if [ "${git_diff_return_code}" != 0 ]; then
+        echo "Failed style check. Download ${CI_JOB_URL}/artifacts/raw/style.patch to see which changes are necessary." >&2
     fi
-    if [ -f "pylint.log" ]; then
-        if [ "${CI}" != "" ]; then
-            errors=$(grep -Pc '^[a-z]+/.+?.py:[0-9]+:[0-9]+: [CRWEF][0-9]+:' pylint.log)
-            maintainer/gh_post_pylint.py "${errors}" pylint.log || exit 1
-            echo "Failed pylint check: ${errors} errors" >&2
-        else
-            echo "Failed pylint check." >&2
-        fi
+    if [ "${pylint_errors}" != 0 ]; then
+        echo "Failed pylint check: ${pylint_errors} errors" >&2
+    fi
+else
+    if [ "${git_diff_return_code}" != 0 ]; then
+        echo "Failed style check." >&2
+    fi
+    if [ "${pylint_errors}" != 0 ]; then
+        echo "Failed pylint check." >&2
     fi
 fi
 

--- a/samples/gibbs_ensemble_socket.py
+++ b/samples/gibbs_ensemble_socket.py
@@ -299,7 +299,6 @@ def check_exchange_particle(boxes, inner_potential, rand_box):
     return True
 
 
-
 # init socket
 s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)

--- a/src/core/particle_data.hpp
+++ b/src/core/particle_data.hpp
@@ -71,9 +71,9 @@ void clear_particle_node();
 /**
  * @brief Get particle data.
  *
- *   @param part the identity of the particle to fetch
- *   @return Pointer to copy of particle if it exists,
- *           nullptr otherwise;
+ *  @param part the identity of the particle to fetch
+ *  @return Pointer to copy of particle if it exists,
+ *          nullptr otherwise;
  */
 const Particle &get_particle_data(int part);
 

--- a/src/python/espressomd/drude_helpers.py
+++ b/src/python/espressomd/drude_helpers.py
@@ -207,7 +207,7 @@ def setup_and_add_drude_exclusion_bonds(system, verbose=False):
     # All Drude types need...
     for td in drude_type_list:
 
-        #...exclusions with core
+        # ...exclusions with core
         qd = drude_dict[td]["q"]  # Drude charge
         qc = drude_dict[td]["qc"]  # Core charge
         subtr_sr_bond = interactions.BondedCoulombSRBond(
@@ -255,9 +255,9 @@ def setup_intramol_exclusion_bonds(system, mol_drude_types, mol_core_types,
     for td in mol_drude_types:
         drude_dict[td]["subtr_sr_bonds_intramol"] = {}
 
-        #... sr exclusion bond with other partial core charges...
+        # ... sr exclusion bond with other partial core charges...
         for tc, qp in zip(mol_core_types, mol_core_partial_charges):
-            #...excluding the Drude core partner
+            # ...excluding the Drude core partner
             if drude_dict[td]["core_type"] != tc:
                 qd = drude_dict[td]["q"]  # Drude charge
                 subtr_sr_bond = interactions.BondedCoulombSRBond(

--- a/src/python/espressomd/interactions.pxd
+++ b/src/python/espressomd/interactions.pxd
@@ -306,7 +306,7 @@ IF TABULATED:
                                  vector[double] force)
 IF ROTATION:
     cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
-        #* Parameters for the harmonic dumbbell bond potential */
+        # Parameters for the harmonic dumbbell bond potential
         cdef struct Harmonic_dumbbell_bond_parameters:
             double k1
             double k2
@@ -320,14 +320,14 @@ ELSE:
         double r_cut
 
 cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
-    #* Parameters for n-body tabulated potential (n=2,3,4). */
+    # Parameters for n-body tabulated potential (n=2,3,4).
     cdef struct Tabulated_bond_parameters:
         int type
         TabulatedPotential * pot
 
 IF ELECTROSTATICS:
     cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
-        #* Parameters for Bonded Coulomb p3m sr */
+        # Parameters for Bonded Coulomb p3m sr
         cdef struct Bonded_coulomb_sr_bond_parameters:
             double q1q2
 ELSE:
@@ -342,14 +342,14 @@ cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
         double drmax2
         double drmax2i
 
-    #* Parameters for oif_global_forces */
+    # Parameters for oif_global_forces
     cdef struct Oif_global_forces_bond_parameters:
         double A0_g
         double ka_g
         double V0
         double kv
 
-    #* Parameters for oif_local_forces */
+    # Parameters for oif_local_forces
     cdef struct Oif_local_forces_bond_parameters:
         double r0
         double ks
@@ -361,13 +361,13 @@ cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
         double kal
         double kvisc
 
-    #* Parameters for harmonic bond Potential */
+    # Parameters for harmonic bond Potential
     cdef struct Harmonic_bond_parameters:
         double k
         double r
         double r_cut
 
-    #* Parameters for thermalized  bond */
+    # Parameters for thermalized  bond
     cdef struct Thermalized_bond_parameters:
         double temp_com
         double gamma_com
@@ -375,35 +375,35 @@ cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
         double gamma_distance
         double r_cut
 
-    #* Parameters for Bonded Coulomb */
+    # Parameters for Bonded Coulomb
     cdef struct Bonded_coulomb_bond_parameters:
         double prefactor
 
-    #* Parameters for three body angular potential (bond_angle_harmonic).
+    # Parameters for three body angular potential (bond_angle_harmonic).
     cdef struct Angle_harmonic_bond_parameters:
         double bend
         double phi0
 
-    #* Parameters for three body angular potential (bond_angle_cosine).
+    # Parameters for three body angular potential (bond_angle_cosine).
     cdef struct Angle_cosine_bond_parameters:
         double bend
         double phi0
         double cos_phi0
         double sin_phi0
 
-    #* Parameters for three body angular potential (bond_angle_cossquare).
+    # Parameters for three body angular potential (bond_angle_cossquare).
     cdef struct Angle_cossquare_bond_parameters:
         double bend
         double phi0
         double cos_phi0
 
-    #* Parameters for four body angular potential (dihedral-angle potentials). */
+    # Parameters for four body angular potential (dihedral-angle potentials).
     cdef struct Dihedral_bond_parameters:
         double mult
         double bend
         double phase
 
-    #* Parameters for n-body overlapped potential (n=2,3,4). */
+    # Parameters for n-body overlapped potential (n=2,3,4).
     cdef struct Overlap_bond_parameters:
         char * filename
         int    type
@@ -413,30 +413,32 @@ cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
         double * para_b
         double * para_c
 
-    #* Parameters for one-directional harmonic potential */
+    # Parameters for one-directional harmonic potential
     cdef struct Umbrella_bond_parameters:
         double k
         int    dir
         double r
 
-    #* Parameters for subt-LJ potential */
+    # Parameters for subt-LJ potential
     cdef struct Subt_lj_bond_parameters:
         double k
         double r
         double r2
 
-    #* Parameters for the rigid_bond/SHAKE/RATTLE ALGORITHM */
+    # Parameters for the rigid_bond/SHAKE/RATTLE ALGORITHM
     cdef struct Rigid_bond_parameters:
-        #*Length of rigid bond/Constrained Bond*/
+        # Length of rigid bond/Constrained Bond
         # double d
-        #*Square of the length of Constrained Bond*/
+        # Square of the length of Constrained Bond
         double d2
-        #*Positional Tolerance/Accuracy value for termination of RATTLE/SHAKE iterations during position corrections*/
+        # Positional Tolerance/Accuracy value for termination of RATTLE/SHAKE
+        # iterations during position corrections
         double p_tol
-        #*Velocity Tolerance/Accuracy for termination of RATTLE/SHAKE iterations during velocity corrections */
+        # Velocity Tolerance/Accuracy for termination of RATTLE/SHAKE
+        # iterations during velocity corrections
         double v_tol
 
-    #* Parameters for IBM Triel */
+    # Parameters for IBM Triel
     cdef cppclass tElasticLaw:
         pass
 
@@ -455,24 +457,25 @@ cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
         double k1
         double k2
 
-    #* Parameters for IBM Tribend */
+    # Parameters for IBM Tribend
     cdef struct IBM_Tribend_Parameters:
         double kb
         double theta0
 
-    #* Parameters for IBM VolCons */
+    # Parameters for IBM VolCons
     cdef struct IBM_VolCons_Parameters:
         int softID
         double kappaV
         double volRef
 
-    #* Parameters for Quartic */
+    # Parameters for Quartic
     cdef struct Quartic_bond_parameters:
         double k0, k1
         double r
         double r_cut
 
-    #* Union in which to store the parameters of an individual bonded interaction */
+    # Union in which to store the parameters of an individual bonded
+    # interaction
     cdef union Bond_parameters:
         Fene_bond_parameters fene
         Oif_global_forces_bond_parameters oif_global_forces
@@ -496,7 +499,7 @@ cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
     cdef struct Bonded_ia_parameters:
         int type
         int num
-        #* union to store the different bonded interaction parameters. */
+        # union to store the different bonded interaction parameters.
         Bond_parameters p
 
     vector[Bonded_ia_parameters] bonded_ia_params

--- a/testsuite/scripts/importlib_wrapper.py
+++ b/testsuite/scripts/importlib_wrapper.py
@@ -440,14 +440,14 @@ def protect_ipython_magics(code):
     formatted comment. This is necessary whenever the code must be parsed
     through ``ast``, because magic commands are not valid Python statements.
     """
-    return re.sub("^(%+)(?=[a-z])", "#_IPYTHON_MAGIC_\g<1>", code, flags=re.M)
+    return re.sub("^(%+)(?=[a-z])", r"#_IPYTHON_MAGIC_\g<1>", code, flags=re.M)
 
 
 def deprotect_ipython_magics(code):
     """
     Reverse the action of :func:`protect_ipython_magics`.
     """
-    return re.sub("^#_IPYTHON_MAGIC_(%+)(?=[a-z])", "\g<1>", code, flags=re.M)
+    return re.sub("^#_IPYTHON_MAGIC_(%+)(?=[a-z])", r"\g<1>", code, flags=re.M)
 
 
 class GetMatplotlibPyplot(ast.NodeVisitor):

